lib/ansible/cli/inventory.py

```diff
@@ -283,7 +283,7 @@ def _graph_group(self, group, depth=0):
         if group.name != 'all':
             for host in sorted(group.hosts, key=attrgetter('name')):
                 result.append(self._graph_name(host.name, depth))
-                result.extend(self._show_vars(host.get_vars(), depth + 1))
+                result.extend(self._show_vars(host._get_host_variables(), depth + 1))
```

Comment:
Do you think we should remove internal variable too. 

```python
                host_vars = self._get_host_variables(host)
                if host_vars:
                    self._remove_internal(host_vars)
                    result.extend(self._show_vars(host_vars, depth + 1))
```

Reply:
good catch